### ISSUE TITLE
Flexible parts

### ISSUE THREAD

**Issue by leozide** [leozide/leocad#658]
- Make it clear somehow in the parts list that a part is flexible
- Write a tutorial on how to bend parts

**Comment by Symbian9** [comment]
> - Make it clear somehow in the parts list that a part is flexible

Maybe there should be just a *"Flexible" category* for all flexible parts?

**Comment by leozide** [comment]
I was thinking an icon in the thumbnails

**Comment by Symbian9** [comment]
> I was thinking an icon in the thumbnails

Yeah, it is expected solution, but "Flexible" category would be useful too.

**Comment by nathaneltitane** [comment]
> I was thinking an icon in the thumbnails

or you could overlay a small 'F' instead in the meantime?

**Comment by Symbian9** [comment]
> or you could overlay a small 'F' instead in the meantime?

Guess `&` or `@` would be better character as a text label for flexible parts.

**Comment by nathaneltitane** [comment]
> > or you could overlay a small `F` instead in the meantime?
> 
> Guess `&` or `@` would be better character as a text label for flexible parts.

K.I.S.S.: F is for 'F'lefxible.

**Comment by Symbian9** [comment]
> K.I.S.S.: F is for 'F'lefxible.

It's not work for other than English locales... (LeoCAD already has 6 locales)

Label should be more symbolic (international).

**Comment by leozide** [comment]
The letter can be marked for translation

**Comment by Symbian9** [comment]
> The letter can be marked for translation

Localization of "`F`lexible" may be not so easy.

For example, in Ukrainian the closest translate would be "`Г`нучкі", but I expect that Ukrainian kids would not recognize what `Г` on top of part thumbnail would mean.

I expect instead to use `&` or even `S` like a symbolic text label as it already recognized by kids around the world as a symbol of 🐍 snake (flexible animal).

**Comment by j6t** [comment]
How about a small light-green cube, or a pair of them, like the control points used to form the flexible parts?

**Comment by Symbian9** [comment]
> How about a small light-green cube, or a pair of them, like the control points used to form the flexible parts?

It would be too complicated to recongnize by kids.

**Comment by rsbx** [comment]
Do kids understand tooltips (hover)?
